src/App/Controllers/Modules/GetContent.php

```diff
@@ -94,11 +94,24 @@ protected function getData($dir)
      */
     protected function getFolderContent($folder, $rec = false)
     {
-        return $this->storageDisk->listWith(
-            ['mimetype', 'visibility', 'timestamp', 'size'],
-            $folder,
-            $rec
-        );
+        return $this->storageDisk->listContents($folder)->map(function ($attributes) {
```

Comment:
plz add `$rec` to the `listContents` check https://flysystem.thephpleague.com/docs/usage/filesystem-api/